libs/ui/src/components/MediaItem/type/VideoMedia.vue

```diff
@@ -1,9 +1,14 @@
 <template>
-  <div class="w-full h-full z-10 min-w-0 relative">
-    <div class="h-0 w-full pb-[100%]" />
+  <div
+    class="h-full min-w-0 relative"
+    :class="{ 'w-[95%]': isFullscreen, 'w-full': !isFullscreen }">
+    <div
+      class="h-0 w-full"
+      :class="{ 'pb-[65%]': isFullscreen, 'pb-[100%]': !isFullscreen }" />
```

Comment:
```suggestion
  <div class="w-full h-full z-10 min-w-0 relative">
    <div class="h-0 w-full pb-[100%]" />
```

It does not need to change now. Could you pls revert it?